src/vaadin-message-input.d.ts

```diff
@@ -0,0 +1,34 @@
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+
+/**
+ * `<vaadin-message-input>` is a Web Component for sending messages.
+ * It consists of a text area that grows on along with the content, and a send button to send message.
+ *
+ * ```html
+ * <vaadin-message-input></vaadin-message-input>
+ * ```
+ *
+ * ### Styling
+ *
+ * The following shadow DOM parts are available for styling:
+ *
+ * Part name | Description
+ * ----------|----------------
+ *
+ * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
```

Comment:
Is the convention to still have this in the docs even if it doesn't have any themable parts?

Reply:
I felt like not deleting it to remind that we should have that info in the docs when we have some themeable parts in the future. I could just as well delete them.

Reply:
If we have plans for adding themable parts to this component, then sounds good. Do we?

Reply:
No plans. I would have guessed that the text-area and button would be stylable parts, but maybe we are doing too much "futureproofing" here. Probably best to remove.

Reply:
Done. Remvoed.

Reply:
Remove then also from this type def file

Reply:
Sorry. Done.